### DISCUSSION TITLE
Geschätzte Leistungsaufnahme für PlanningRequest

### DISCUSSION THREAD

**Discussion by BenSahlmann** [camueller/SmartApplianceEnabler#228] [Allgemein]
Moin Axel,

ich hatte den SAE jahrelang in einer frühen Version im Einsatz, um PV-Anhebung mit einer Brauchwasserwärmepumpe zu machen. Hierfür hatte ich ein Hauptdevice und ein SubDevice konfiguriert, wobei nur das Hauptdevice einen S0-Zähler hatte und nur das SubDevice einen Schalter. Irgendwie hatte ich es in einer frühen Version hinbekommen, dass der SAE dem SHM nur für den PlanningRequest eine geschätzte Leistungsaufnahme mitteilt - damit kam dann die Einplanung zurück und der echte Verbrauch wurde über den S0 der Hauptdevices korrekt erfasst. Diese Funktionalität scheint aber mit einem Update unterwegs verlorengegangen zu sein, denn der SAE meldet dem SHM nun den "wahren" Verbrauch (0 Watt) eines nicht angeschlossenen S0-Zählers womit der SHM das Gerät niemals einplant. Eventuell lässt sich diese Funktionalität wieder herstellen - alternativ bin ich auch beim Master-Slave-Konzept dabei (ich arbeite hier ebenfalls mit einem SubDevice, damit das Gerät nur einmal im Portal auftaucht), da wird aber derzeit der S0-Zähler nicht gezählt (alle anderen schon).

Danke für deine Rückmeldung
Ben

**Comment by camueller** [comment]
Ich habe gerade mal im Code nachgesehen: Wenn ein Gerät keinen Zähler konfiguriert hat, wird die konfigurierte, maximale Leistungsaufnahme gemeldet WENN das Gerät eingeschaltet ist.

Ansonsten freue ich mich über jeden, der Master-Slave-Zähler testet. Allerdings sind da aktuell Fehler drin, die ich inzwischen behoben habe, aber noch kein Release gemacht habe. Es wird aber demnächst (ca. 1 Woche) ein Release geben.
